samples/samples-outofproc/Microsoft.Azure.WebJobs.Extensions.Sql.SamplesOutOfProc.csproj

```diff
@@ -11,6 +11,7 @@
     <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.Timer" />
     <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.Storage.Queues" />
     <PackageReference Include="Microsoft.AspNetCore.Http" />
+    <PackageReference Include="Grpc.Net.Client" VersionOverride="2.55.0" />
```

Comment:
Do you need to use VersionOverride? You should just be able to add it to https://github.com/Azure/azure-functions-sql-extension/blob/main/Directory.Packages.props and then reference it directly here

Reply:
Yes I believe I do and I tried doing what you mentioned first; please take a look at this branch where I tried doing that: https://github.com/Azure/azure-functions-sql-extension/compare/chlafren/directorypacks?expand=1

https://github.com/Azure/azure-functions-sql-extension/compare/chlafren/directorypacks?expand=1#diff-2b567884a3a252859000ac56f9a9caed8eab3d1ce97addce0cc4996a7f43ee02R839

Reply:
you still need to add the package reference in the two csproj's. Once you do that it should pick up the specified version

Reply:
(you just won't specify the version when doing so)

Reply:
ahhh that's why i was so confused. ok on it!